### PR TITLE
implemented question component

### PR DESCRIPTION
**Pull request by agentanky**
Created question component with hardcoded / sample data.  Please review, team!

### CODE REVIEW

**Review by phil-ho — APPROVED** [review]
nice work!

File: data/answers.js [hunk 1]
```diff
@@ -90,7 +90,9 @@
...
-              }
             }
-  ]
-};
\ No newline at end of file
+        }
+    ]
+};
+
+export default answers;
```
Comment (phil-ho) [inline data/answers.js:98]: actually this export exports something called answers, but there's no answers variable defined in this file.  Perhaps at the top set the object to...
```suggestion
const answers = { ...
```

File: data/answers.js [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-              },
-              "800555": {
-                  "id": 800555,
-                  "body": "Pariatur accusamus vitae sit in corrupti dolor ut ab.",
-                  "date": "2020-09-21T00:00:00.000Z",
-                  "answerer_name": "Kelvin8",
-                  "helpfulness": 16,
-                  "photos": []
-              }
+    {
```
Comment (phil-ho) [inline data/answers.js:1]:
```suggestion
    const answers = {
```

**Review by moneycabui — APPROVED** [review]
Great job Ankur!

**Pull request merged**